src/index.js

```diff
@@ -0,0 +1,237 @@
+
+function updateOutputCity(event) {
+  const outputCityElement = document.getElementById('outputCity');
+  // state.city = event.target.value;
+  state.city = document.getElementById('inputCity').value;
+  outputCityElement.textContent = state.city;
+}
+
+// Function to reset the output city by clicking reset button
+// const resetCityAction = () => {
+//   // const cityResetElement = document.getElementById('cityReset');
+//   const inputCityElement = document.getElementById('inputCity');
+//   inputCityElement.value = 'Orlando';
+
+//   const outputCityElement = document.getElementById('outputCity');
+//   outputCityElement.textContent = 'Orlando';
+//   state.city = 'Orlando';
+// };
+
+const resetCityAction = () => {
+  state.city = 'Orlando';
+  const inputCityElement = document.getElementById('inputCity');
+  inputCityElement.value = state.city;
+  updateOutputCity();
+  getLatLon();
+};
+
+const registerEventHandler = () => {
+  // event listener for clicking up arrow
+  const upTempElement = document.getElementById('upTemp');
+  upTempElement.addEventListener('click', incTempAction);
+
+  // event listener for clicking down arrow
+  const downTempElement = document.getElementById('downTemp');
+  downTempElement.addEventListener('click', decTempAction);
+
+  // event listener for dropdown so sky can change
+  const skyDropdownElement = document.getElementById('skyDropdown');
+  skyDropdownElement.addEventListener('change', updateSkyAction);
+
+  // event listener for inputting city so can display city
+  const inputCityElement = document.querySelector('input');
+  const outputCityElement = document.getElementById('outputCity');
+  inputCityElement.addEventListener('input', updateOutputCity);
+  inputCityElement.addEventListener('change', getLatLon);
+
+  // event listener for clicking city reset button
+  const cityResetElement = document.getElementById('cityReset');
+  cityResetElement.addEventListener('click', resetCityAction);
+
+  const realTimeTempElement = document.getElementById('realTimeTemp');
+  realTimeTempElement.addEventListener('click', getRealTemp);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandler);
+
+/* <label>Choose an ice cream flavor:
+  <select class="ice-cream" name="ice-cream">
+    <option value="">Select One …</option>
+    <option value="chocolate">Chocolate</option>
+    <option value="sardine">Sardine</option>
+    <option value="vanilla">Vanilla</option>
+  </select>
+</label>
+
+<div class="result"></div>
+
+JavaScript
+const selectElement = document.querySelector('.ice-cream');
+
+selectElement.addEventListener('change', (event) => {
+  const result = document.querySelector('.result');
+  result.textContent = `You like ${event.target.value}`;
+}); */
+
+// const arrowEx = () => {
+//   // state.arrow += "⬆"
+//   const upTempElement = document.getElementById('upTemp');
+//   upTempElement.textContent += '⬆';
+// };
+
+// upTempElement.addEventListener('click', arrowEx);
```

Comment:
Be sure to clean up your code to remove comments/print statements, which makes your code more readable.